docs/architecture/cloud-native/toc.yml

```diff
@@ -46,10 +46,14 @@
   - name: Distributed Data
     href: distributed-data.md
```

Comment:
you removed this article. This TOC needs to be changed then with the new structure.

Reply:
Done